subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/logcat/BufferLogcatAccessor.kt

```diff
@@ -0,0 +1,16 @@
+package com.avito.instrumentation.internal.logcat
+
+import com.avito.android.Problem
+
+internal class BufferLogcatAccessor(private val logcatBuffer: LogcatBuffer?) : LogcatAccessor {
```

Comment:
We could check `logcatBuffers.get(key)` and if there is `null` then call `device.logcat()`